app/actions/LendingRequestActions.ts

```diff
@@ -0,0 +1,61 @@
+import { LendingRequest } from 'app/actions/ActionTypes';
+import callAPI from 'app/actions/callAPI';
+import { lendingRequestSchema } from 'app/reducers';
+import type {
+  EntityType,
+  NormalizedEntityPayload,
+} from 'app/store/models/entities';
+import type { Thunk } from 'app/types';
+
+export function fetchAllLendingRequests(): Thunk<
+  Promise<NormalizedEntityPayload<EntityType.LendingRequests>>
+> {
+  return callAPI({
+    types: LendingRequest.FETCH,
+    endpoint: '/lendinginstance/',
+    schema: [lendingRequestSchema],
+    meta: {
+      errorMessage: 'Henting av utlånsforespørsler failet',
+    },
+    propagateError: true,
+  });
+}
```

Comment:
```suggestion
import type { LendingRequest } from app/store/models/LendingRequest.ts;

export function fetchAllLendingRequests() {
  return callAPI<LendingRequest>({
    types: LendingRequest.FETCH,
    endpoint: '/lendinginstance/',
    schema: [lendingRequestSchema],
    meta: {
      errorMessage: 'Henting av utlånsforespørsler feilet',
    },
    propagateError: true,
  });
}
```

This applies to all actions here